### DISCUSSION TITLE
Thank you for our first 1000 stars on GitHub!

### DISCUSSION THREAD

**Discussion by AnonymousWP** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#418] [Announcements]
From the entire team, we thank you for getting us [1000 stars on GitHub](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/stargazers).

Since 0x192 has sort of abandoned the project, which can be noticed by the incredible amount of open PRs, we decided to fork the project. Also because we believe such projects shouldn't be managed by 1 user with write rights. In our org, there are two owners and 4 other contributors/members who have write access. This makes sure that the project never depends on just 1 person.

Speaking of contributors: we need both [application contributors](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/blob/main/CONTRIBUTING.md) and [package contributors](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/wiki/How-to-contribute). There are still [enough issues to work on](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).

**Comment by Rudxain** [comment]
Road to 2k! 🎉
